src/odfesqlodbc/es_communication.h

```diff
@@ -39,8 +40,36 @@
 #include <aws/core/http/HttpClientFactory.h>
 #include <aws/core/http/HttpClient.h>
 #include <aws/core/client/ClientConfiguration.h>
+#include <mutex>
+#include <condition_variable>
+#include <deque>
+
 // clang-format on
 
+template < typename T >
+class BlockingQueue {
```

Comment:
This looks like a straight up copy of https://stackoverflow.com/questions/12805041/c-equivalent-to-javas-blockingqueue - is it?

Reply:
removed